R/deltaccd_plot.R

```diff
@@ -103,9 +115,9 @@ plotHeatmap = function(geneNames, emat, groupVec = NULL) {
   } else if (min(table(groupVec)) < 3) {
     stop('Each unique group in groupVec must have at least three samples.')}
 
-  df = calcCorr(ematNow, groupVec, method)
-  cLims = calcColorLimits(df$rho)
-  p = plotHeatmapSimple(ggplot2::ggplot(df) + ggplot2::facet_wrap(~ group),
+  dt = calcCorr(ematNow, groupVec, method)
+  cLims = calcColorLimits(dt$rho)
+  p = plotHeatmapSimple(ggplot2::ggplot(dt) + ggplot2::facet_wrap(~ group),
```

Comment:
While we're at it, we might as well update the call to facet_wrap to use the currently recommended syntax, which uses `vars` instead of a formula.